src/modules/Channel/components/Message/index.tsx

```diff
@@ -209,6 +211,10 @@ const Message = ({
     return null;
   }, [message, renderCustomSeparator]);
 
+  const suggestedReplies = useMemo(() => {
+    return message.extendedMessagePayload?.suggested_replies as string[] | undefined ?? [];
+  }, [(message.extendedMessagePayload?.suggested_replies as string[] | undefined)?.length]);
```

Comment:
The `extendedMessagePayload` will be included in the upcoming JS chat SDK. So I need to push a commit to bump up the chat sdk version once it's release.

Reply:
If it's a payload that is included in a specific message, it seems like there won't be significant changes. Should we consider removing the useMemo?

Reply:
Yeah it makes sense. Just addressed it https://github.com/sendbird/sendbird-uikit-react/pull/825/commits/13206b3337445d1f7c1673aac7be6542674ea808 :)